example/ios/example/AppDelegate.mm

```diff
@@ -3,7 +3,8 @@
 #import <React/RCTBundleURLProvider.h>
 #import <React/RCTLinkingManager.h>
 
-#import <RNNotifications.h>
```

Comment:
this approach and below is not ideal. i tried (only for a short time) to make a config file and opt-in that way but wasnt having much luck so decided to put the PR up for review and collect feedback from you guys on a potentially better way